packages/relay/src/lib/services/index.ts

```diff
@@ -2,9 +2,23 @@
 import { BlockService } from './blockService/BlockService';
 import { IBlockService } from './blockService/IBlockService';
 import { ContractService } from './contractService/ContractService';
+import { IContractService } from './contractService/IContractService';
 import { CommonService } from './ethService/ethCommonService/CommonService';
 import { ICommonService } from './ethService/ethCommonService/ICommonService';
 import { FilterService } from './ethService/ethFilterService';
 import { IFilterService } from './ethService/ethFilterService/IFilterService';
+import MetricService from './metricService/metricService';
+import { TransactionService } from './transactionService/TransactionService';
 
-export { BlockService, IBlockService, ICommonService, CommonService, IFilterService, FilterService, ContractService };
+export {
+  BlockService,
+  IBlockService,
+  ICommonService,
+  CommonService,
+  IFilterService,
+  FilterService,
+  IContractService,
+  ContractService,
+  TransactionService,
+  MetricService,
+};
```

Comment:
Can we do something like below for cleaner code?

```suggestion
export * from './blockService/BlockService'
export * from './blockService/IBlockService'
export * from './contractService/ContractService'
export * from './contractService/IContractService'
export * from './ethService/ethCommonService/CommonService'
export * from './ethService/ethCommonService/ICommonService'
export * from './ethService/ethFilterService'
export * from './ethService/ethFilterService/IFilterService'
```

Reply:
yeah, will change it